src/app/lecture/lecture.component.html

```diff
@@ -0,0 +1,3 @@
+<div id="lecture">
+  <cc-knowledge-node *ngFor="let knowledgeNode of knowledgeNodes" [learningObjects]="knowledgeNode.learningObjects"></cc-knowledge-node>
```

Comment:
I am not quite sure yet if the better option would be to have a KN class contain an array of KNs that it unlocks, or an array of KNs that are its prerequisite. So far I've gone with the first option, though this might change depending on our needs.